view/frontend/layout/catalog_category_view.xml

```diff
@@ -6,4 +6,5 @@
             <block class="Tutorialstab\LazyloadingInfiniteScroll\Block\Init" name="lazyloadinginfinitescroll.scripts" template="Tutorialstab_LazyloadingInfiniteScroll::scripts.phtml" />
```

Comment:
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting.